repository/CargoPackageManager-Minimal.package/CGODirectoryPackageRepository.class/class/newInDirectory..st

```diff
@@ -0,0 +1,7 @@
+instance creation
+newInDirectory: aPathOrFileReference
```

Comment:
Yes, except by the fact that #newWithFileName: will force the file to be in the real file system, because it does this:

^ self newWithUrl: aFileName asFileReference fullName

I did not simply remove the old method to keep compatibility so far.